core/src/main/java/foundation/stack/datamill/configuration/Wiring.java

```diff
@@ -422,9 +422,11 @@ public Wiring addFormatted(String name, String format, Object... arguments) {
                     try {
                         values[i] = construct(parameterType, autoConstructionCandidates);
                     } catch (IllegalArgumentException | IllegalStateException e) {
+                        logger.error("Could not build class {} as the following type could not be constructed {}", clazz, parameterType, e);
```

Comment:
I think these should be debug statements, without logging stack traces - otherwise, this is going to clutter logs for what is a normal case - when you call construct(Class<?>), it goes through all constructors - so you'll see a lot of errors for cases where it succeeded with another constructor